examples/story_diffusion/utils/utils.py

```diff
@@ -0,0 +1,464 @@
+import base64
+import random
+from io import BytesIO
+
+import gradio as gr
+import numpy as np
+from PIL import Image, ImageDraw, ImageOps
+
+import mindspore as ms
+from mindspore import ops
+
+MAX_COLORS = 12
+
+
+def get_random_bool():
+    return random.choice([True, False])
+
+
+def add_white_border(input_image, border_width=10):
+    """
+    为PIL图像添加指定宽度的白色边框。
```

Comment:
Same here and the rest

Reply:
Some legacy comments copied from original repo were Chinese. I have deleted them because they are not important for understanding.